src/pages/library/common.js

```diff
@@ -77,7 +81,7 @@ const FilterSection = ({ onTagFilter, labels, selectedTags, labelRenderer, listD
         onClick: () => onTagFilter({ lowerTag, label })
       }, [
         div({ style: { lineHeight: '1.375rem', flex: 1 } }, [...(labelRenderer ? labelRenderer(label) : label)]),
-        div({ style: styles.pill(_.includes(lowerTag, lowerSelectedTags)) }, [_.size(listDataByTag[lowerTag])])
+        div({ 'aria-label': `${numMatches} matches`, style: styles.pill(isChecked) }, [numMatches])
```

Comment:
Added a label for a better reader experience. When navigating the page, the reader will announce (as an example):

"Epigenomic 4 matches unchecked"
rather than 
"Epigenomic 4 unchecked"

Reply:
this makes sense!